packages/expo-notifications/src/BadgeModule.web.ts

```diff
@@ -1,16 +1,20 @@
-import * as badgin from 'badgin';
-
 import { BadgeModule } from './BadgeModule.types';
 
 let lastSetBadgeCount = 0;
 
-export default {
+const badgeModule: BadgeModule = {
   addListener: () => {},
   removeListeners: () => {},
   getBadgeCountAsync: async () => {
     return lastSetBadgeCount;
   },
-  setBadgeCountAsync: async (badgeCount: number, options?: badgin.Options) => {
+  setBadgeCountAsync: async (badgeCount, options) => {
+    // If this module is loaded in SSR (NextJS), we can't modify the badge.
+    // It also can't load the badgin module, that instantly invokes methods on window.
+    if (typeof window === 'undefined') {
+      return false;
+    }
+    const badgin = require('badgin');
```

Comment:
Maybe we could even `await import()`? 🤔 Would it work in SSR?

Reply:
I can try 😄

Reply:
I'm afraid not:

![Screenshot 2020-10-20 at 14 26 04](https://user-images.githubusercontent.com/1203991/96585698-3cfb2b80-12e0-11eb-87ac-95520d04b021.png)